src/messages/layouts/MessageLayoutContainer.cpp

```diff
@@ -390,17 +390,17 @@ void MessageLayoutContainer::paintSelection(QPainter &painter, int messageIndex,
                         int lineIndex2 = lineIndex + 1;
                         for (; lineIndex2 < this->lines_.size(); lineIndex2++)
                         {
-                            Line &line = this->lines_[lineIndex2];

```

Comment:
Same applies here as above

Reply:
This name is more relevant to the `lineIndex2` variable.

Reply:
sure